src/Sarif.Viewer.VisualStudio/ErrorList/SarifSnapshot.cs

```diff
@@ -188,26 +163,31 @@ private void ErrorListInlineLink_Click(object sender, RoutedEventArgs e)
 
             if (hyperLink != null)
             {
-                Tuple<int, int> indices = hyperLink.Tag as Tuple<int, int>;
+                Tuple<object, int> data = hyperLink.Tag as Tuple<object, int>;
```

Comment:
To expand on a comment I made elsewhere: Why store this index as an `object`? As the code stands you have to convert the integer index to `object` on the way in, and then call `Convert.ToInt32` on the way out. Why not go with `int` all the way?